mobile/test/common/integration/test_server.h

```diff
@@ -23,6 +23,9 @@ class TestServer {
   Thread::SkipAsserts skip_asserts_;
   ProcessWide process_wide;
   Thread::MutexBasicLockable lock;
+  std::string header_key_;
+  std::string header_value_;
+  std::string data_;
```

Comment:
Are these 3 new members used? I'm not sure I see where.

Reply:
Ah good catch stopped using them yeah